tests/test_losses/test_likelihood.py

```diff
@@ -0,0 +1,74 @@
+# Black-box ABM Calibration Kit (Black-it)
+# Copyright (C) 2021-2022 Banca d'Italia
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""This module contains tests for the Likelihood loss."""
+import numpy as np
+
+from black_it.loss_functions.likelihood import LikelihoodLoss
+
+np.random.seed(3)
```

Comment:
this is a bit dangerous as the line is executed at module level, but it is not guaranteed the seeds, across tests, are set in the same order because of changes in the order in how the test modules are imported. I suggest using a different approach; how is it done in other tests?

Reply:
ok I see each test calls `np.random.seed` in the body of each test. I argue against this practice as `np.random.seed` does a global side-effect and might cause dependency among tests. This should be refactored in using a local random number generator (`np.default_rng(seed)`) whenever possible. 

For this PR, I suggest moving this call within the body of each test.

Reply:
I don't think we have a test with the same issue

Reply:
I can set the seeds within tests. In any case, the tests should work across different seeds